### PR TITLE
Add commands

### PR DESCRIPTION
**Pull request by BenSanex**

### CODE REVIEW

**Comment by BenSanex** [comment]
Hi, learning about comments!

**Comment by tbogin** [comment]
Here I am

File: README.md [hunk 1]
```diff
@@ -1 +1,8 @@
-# phase-0-gps-1
\ No newline at end of file
+# phase-0-gps-1
+```
+git clone
```
Comment (tbogin) [inline README.md:3]: Clone the repo

File: README.md [hunk 2]
```diff
@@ -1 +1,8 @@
...
+git add
+git commit -m
+git push
+git checkout -b
+```
```
Comment (BenSanex) [inline README.md]: Don't forget git status!

**Pull request merged**